docker/build-trigger.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Triggers an auto-build on dockerhub for the given source control reference.
+#
+# Example usage: ./build-trigger Tag 1.0.0 https://registry.hub.docker.com/u/scitran/reaper/trigger/11111111-2222-3333-4444-abcdefabcdef/
+function Main() {(
```

Comment:
What's the purpose of the extra parenthesis around the function body? Is that to force a subshell? Which you then export your local variables out of?